### PR TITLE
Port to grafana toolkit

### PR DESCRIPTION
**Pull request by srclosson**

### CODE REVIEW

File: yarn-error.log [hunk 1]
```diff
@@ -0,0 +1,11406 @@
+Arguments: 
+  /home/stephanie/.nvm/versions/node/v12.15.0/bin/node /home/stephanie/.nvm/versions/node/v12.15.0/bin/yarn install
+
+PATH: 
+  /home/stephanie/.gvm/pkgsets/go1.13.6/global/bin:/home/stephanie/.gvm/gos/go1.13.6/bin:/home/stephanie/.gvm/pkgsets/go1.13.6/global/overlay/bin:/home/stephanie/.gvm/bin:/home/stephanie/.gvm/bin:/home/stephanie/.nvm/versions/node/v12.15.0/bin:/home/stephanie/.vscode-server/bin/ae08d5460b5a45169385ff3fd44208f431992451/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```
Comment (ryantxu) [inline yarn-error.log]: yarn-error shoudl not be included

File: package.json [hunk 2]
```diff
@@ -3,45 +3,23 @@
   "version": "2.0.1",
   "description": "Grafana data source for Azure Data Explorer",
   "scripts": {
-    "build": "node --max-old-space-size=8192 ./node_modules/.bin/webpack --config webpack.config.prod.js",
-    "dev": "webpack --mode development",
-    "watch": "webpack --mode development --watch",
-    "test": "jest --config jest.config.js",
-    "test-watch": "jest --watch --config jest.config.js"
+    "build": "grafana-toolkit plugin:build && find src | grep '\\.d\\.ts$' | xargs rm -fv",
```
Comment (stevenvachon) [inline package.json]: Why the find/grep/xargs, and what do they do?

File: src/query_builder.test.ts [hunk 3]
```diff
@@ -1,15 +1,15 @@
 import QueryBuilder from './query_builder';
-import moment from 'moment';
 
 describe('QueryBuilder', () => {
-  let builder: QueryBuilder;
+  let builder: QueryBuilder; // tslint:disable-line
```
Comment (stevenvachon) [inline src/query_builder.test.ts]: There is no tslint

File: src/request_aggregator.test.ts [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import RequestAggregator from './request_aggregator';
 
 describe('RequestAggregator', () => {
-  let requestAggregator: RequestAggregator;
+  let requestAggregator: RequestAggregator; // tslint:disable-line
```
Comment (stevenvachon) [inline src/request_aggregator.test.ts]: no tslint

File: src/version.test.ts [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import { SemVersion, isVersionGtOrEq } from './version';
 
 describe('SemVersion', () => {
-  let version = '1.0.0-alpha.1';
+  let version = '1.0.0-alpha.1'; // tslint:disable-line
```
Comment (stevenvachon) [inline src/version.test.ts]: no tslint

File: tsconfig.json [hunk 6]
```diff
@@ -2,10 +2,9 @@
   "extends": "./node_modules/@grafana/toolkit/src/config/tsconfig.plugin.json",
```
Comment (stevenvachon) [inline tsconfig.json]: you could use `"@grafana/toolkit/..."`

File: src/annotations_query_ctrl.ts [hunk 7]
```diff
@@ -7,7 +7,7 @@ export class KustoDBAnnotationsQueryCtrl {
 
   defaultQuery = '<your table>\n| where $__timeFilter() \n| project TimeGenerated, Text=YourTitleColumn, Tags="tag1,tag2"';
 
-  /** @ngInject **/
+  /** @ngInject */
```
Comment (scottlepp) [inline src/annotations_query_ctrl.ts]: I think it's:  /* @ngInject */

File: package.json (hunk 2 above)
Comment (srclosson) [reply package.json]: Yeah, about that... The toolkit leaves a bunch of .d.ts files in the incoming folder... on the canary version anyway. ???

File: src/annotations_query_ctrl.ts (hunk 7 above)
Comment (srclosson) [reply src/annotations_query_ctrl.ts]: Are you sure? https://github.com/search?q=org%3Agrafana+%2F**+ngInject+*%2F&type=Code

Comment (scottlepp) [reply src/annotations_query_ctrl.ts]: I guess it doesn't matter.  I've always used it like the ng-annotate docs suggest:  https://www.npmjs.com/package/babel-plugin-angularjs-annotate?activeTab=readme

File: package.json (hunk 2 above)
Comment (stevenvachon) [reply package.json]: Hmm, weird, that didn't happen in the perspective-panel repo, and it's tsconfig is mostly the same as yours.

**Pull request merged**